build.gradle

```diff
@@ -26,17 +33,13 @@ repositories {
     }
 }
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '2.5'
-}
-
 configurations {
     codeMetrics
 }
 
 dependencies {
 
-    codeMetrics(group: 'org.terasology.config', name: 'codemetrics', version: '1.0.0', ext: 'zip')
+    codeMetrics(group: 'org.terasology.config', name: 'codemetrics', version: '1.3.2', ext: 'zip')
```

Comment:
We're at 1.5.0 now (for shorter license headers).

Reply:
Same here, already addressed - I guess I forgot that I did that, and thought I still had to, heh.

Reply:
Oddly, I am ending up leaving this on just 1.4.0 anyway.. Seems to fail over some regex pattern on 1.5.0